.buildkite/pipeline-utils/ci-stats/pick_test_group_run_order.ts

```diff
@@ -124,15 +127,53 @@ function isObj(x: unknown): x is Record<string, unknown> {
   return typeof x === 'object' && x !== null;
 }
 
+interface PartialConfigs {
+  enabled?: string[];
+  disabled?: string[];
+  defaultQueue?: string;
+}
+
 function getEnabledFtrConfigs(patterns?: string[]) {
+  const configs: {
+    enabled: Array<string | { [configPath: string]: { queue: string } }>;
+    uniqueQueues: Set<string>;
+    defaultQueue?: string;
+  } = {
+    enabled: [],
+    uniqueQueues: new Set<string>(),
```

Comment:
When we split 1 file into multiple, in theory every manifest has its own `defaultQueue`.
Afaik our dynamic pipeline does not support it and I'm not sure we need to add support in this PR. At the moment I added a check to verify that we don't have different `defaultQueue` coming from different manifests and throw error, if it happens. Thoughts?